configs/tsup/src/index.ts

```diff
@@ -10,3 +10,12 @@ export const options: Options = {
   sourcemap: true,
   dts: true,
 }
+
+export const scriptOptions: Options = {
+  clean: true,
+  format: ['cjs'],
+  target: ['chrome51', 'firefox53', 'edge18', 'safari11', 'ios11', 'opera38', 'es6', 'node14'],
+  entry: ['src/scripts/**/*.{ts,tsx}', '!**/*.{spec,test,test-d,bench}.*'],
```

Comment:
I think we should remove this `'src/scripts/**/*.{ts,tsx}'`. check out please!

```suggestion
  entry: ['src/scripts/*.{ts,tsx}', '!**/*.{spec,test,test-d,bench}.*'],
```

Reply:
> I think we should remove this `'src/scripts/**/*.{ts,tsx}'`. check this out please!

That's a good point! There's no need to build the parent directory (scripts/utils) that is only used by scripts and not by the user.